src/rules/__tests__/utils.test.ts

```diff
@@ -1,12 +1,35 @@
+import { JSONSchemaForNPMPackageJsonFiles } from '@schemastore/package';
 import { TSESLint } from '@typescript-eslint/utils';
+import dedent from 'dedent';
 import {
   createRule,
   getNodeName,
   isDescribeCall,
+  isHookCall,
   isTestCaseCall,
 } from '../utils';
 import { espreeParser } from './test-utils';
 
+const findESLintVersion = (): number => {
+  try {
+    const eslintPath = require.resolve('eslint/package.json');
+
+    const eslintPackageJson =
+      // eslint-disable-next-line @typescript-eslint/no-require-imports
+      require(eslintPath) as JSONSchemaForNPMPackageJsonFiles;
+
+    if (eslintPackageJson.version) {
+      const [majorVersion] = eslintPackageJson.version.split('.');
+
+      return parseInt(majorVersion, 10);
+    }
+  } catch {}
```

Comment:
would be nice to just remove the `try` - if something inside here fails, that error is useful in debugging